roles/openshift_master/tasks/main.yml

```diff
@@ -87,6 +87,9 @@
       master_count: "{{ openshift_master_count | default(None) }}"
       controller_lease_ttl: "{{ osm_controller_lease_ttl | default(None) }}"
       master_image: "{{ osm_image | default(None) }}"
+    role: common
+    local_facts:
+      imageconfig_latest: "{{ openshift_imageconfig_latest | default(False) }}"
```

Comment:
The common role addition will need to be separated into a new openshift_facts task or this task can be modified to iterate over the roles like https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-master/config.yml#L34-L59